src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 198px;
+  @include border(1px, solid, $border-color, 5px);
+  .card__item {
```

Comment:
Such nesting is unnecessary and increases the specificity of the selector which is generally bad. Use SCSS nesting instead.

Reply:
> Such nesting is unnecessary and increases the specificity of the selector which is generally bad. Use SCSS nesting instead.

If I nest like this .card__item inside .card it is not scss nesting??

Reply:
> Such nesting is unnecessary and increases the specificity of the selector which is generally bad. Use SCSS nesting instead.

Done